docs/manifest.json

```diff
@@ -30,8 +30,14 @@
       "root": "guides"
     },
     [
-      ["Implement basic Role Based Access Control with metadata", "/guides/basic-rbac"],
-      ["Hide Personal Accounts and force organizations", "/guides/force-organizations"]
+      [
+        "Implement basic Role Based Access Control with metadata",
+        "/guides/basic-rbac"
+      ],
+      [
+        "Hide Personal Accounts and force organizations",
+        "/guides/force-organizations"
+      ]
```

Comment:
Can you remove the formatting changes from this file? They'll clutter the git blame and can lead to merge conflicts with `main` if any of these things are changed.

We'll need to fix consistent formatting independent of editor setup in another PR.

Reply:
I'm not sure why they are present, to be honest. I didn't enable any sort of formatter, and I am not sure how to remove them 😕